game_init.pl

```diff
@@ -86,13 +85,13 @@
     add_achievement("Kill the dragon."),
     retractall(event(dragon_dead, _)),
     println("You've killed the dragon. You're a real hero!"),
+    add_killing_achiev,
```

Comment:
to nie jest jedyne miejsce gdzie gra się kończy, można jeszcze ukończyć poprzez przemianę smoka